x-pack/plugins/observability/public/pages/alert_details/index.tsx

```diff
@@ -7,23 +7,47 @@
 
 import React from 'react';
 import { i18n } from '@kbn/i18n';
-import { AlertSummary } from './components';
+import { useParams } from 'react-router-dom';
+import { noop } from 'lodash';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+
 import { useKibana } from '../../utils/kibana_react';
 import { ObservabilityAppServices } from '../../application/types';
 import { usePluginContext } from '../../hooks/use_plugin_context';
 import { useBreadcrumbs } from '../../hooks/use_breadcrumbs';
 import { paths } from '../../config/paths';
+import { AlertSummary, PageTitle } from './components';
+import { HeaderActions } from './components/header_actions';
+import { Alert, AlertDetailsPathParams } from './types';
 import PageNotFound from '../404';
 
-// import { useParams } from 'react-router';
-// import { AlertDetailsPathParams } from './types';
-
 export function AlertDetailsPage() {
   const { http } = useKibana<ObservabilityAppServices>().services;
 
   const { ObservabilityPageTemplate, config } = usePluginContext();
-  // const { alertId } = useParams<AlertDetailsPathParams>();
-  const alert = {};
+  const { alertId } = useParams<AlertDetailsPathParams>();
+
+  const alert: Alert = {
```

Comment:
@CoenWarmer, I would suggest removing the dummy data as it will require another PR to complete this PR. Besides, you are already providing a storybook if we want to see how the component will render the data.